linera-io/linera-protocol
# Persist Prometheus data

## Motivation

`prometheus-operator` by default stores the data on a `emptyDir` which is not persisted on restarts

## Proposal

Persist the Prometheus data

## Test Plan

1. Checked inside the Docker container where the Prometheus DB will be mounted that the data is there
2. Restart the prometheus pod
3. See that previous data is still there, as well as new data
